README.txt

```diff
@@ -217,7 +217,7 @@ Options
 
    Files, directories, function names, and class names that match this
    regular expression are considered tests.  Default:
-   (?:^|[b_./-])[Tt]est [NOSE_TESTMATCH]
+   (?:\b|_)[Tt]est [NOSE_TESTMATCH]
```

Comment:
When I run that, it comes out as:

    Files, directories, function names, and class names that match this
    regular expression are considered tests.  Default: (?:b|_)[Tt]est
    [NOSE_TESTMATCH]

i.e., the backslash is missing.  Is that a Sphinx bug?

Maybe that's why the original had doubled backslashes -- they were trying to get it to show up correctly in the README.  Even with 4 backslashes, I can't get it to render a backslash in README.txt, though.